Creation of sriov multus cr configuration via spidermultus failed.

**Describe the version**
version about:
1. spiderpool v0.6.0


**Describe the bug**
After creating spidermultus, the spidercontroller panics

**How To Reproduce**

```
apiVersion: spiderpool.spidernet.io/v2beta1
kind: SpiderMultusConfig
metadata:
  creationTimestamp: "2023-08-24T07:11:45Z"
  generation: 1
  name: sriov-3f7640eab8
  namespace: ns-531b808b2d
  resourceVersion: "42080"
  uid: 178f77b8-afde-40e3-abab-161f7a706c21
spec:
  cniType: sriov
  enableCoordinator: true
  sriov:
    resourceName: sriov-test

~# kubectl get spidermultusconfigs.spiderpool.spidernet.io -A |grep sriov                             
ns-531b808b2d   sriov-3f7640eab8          41m
~# kubectl get network-attachment-definitions.k8s.cni.cncf.io -A                                      
NAMESPACE       NAME                      AGE
kube-system     macvlan-vlan0             4h2m
kube-system     macvlan-vlan100           4h2m
kube-system     macvlan-vlan200           4h2m
ns-6610544d08   invalid-json-e8734e57da   156m
```
error logs：
```
{"level":"INFO","ts":"2023-08-24T07:50:14.155Z","logger":"Subnet-Informer","caller":"subnetmanager/subnet_informer.go:294","msg":"Succeed to SYNC","SubnetName":"vlan100-v4","Operation":"PROCESS"}
E0824 07:50:14.305147       1 runtime.go:79] Observed a panic: "assignment to entry in nil map" (assignment to entry in nil map)
{"level":"INFO","ts":"2023-08-24T07:50:14.304Z","logger":"Subnet-Informer","caller":"subnetmanager/subnet_informer.go:294","msg":"Succeed to SYNC","SubnetName":"vlan200-v4","Operation":"PROCESS"}
goroutine 441 [running]:
k8s.io/apimachinery/pkg/util/runtime.logPanic({0x1bcca80?, 0x214d140})
        /src/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:75 +0x99
k8s.io/apimachinery/pkg/util/runtime.HandleCrash({0x0, 0x0, 0xc000650a50?})
        /src/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:49 +0x75
panic({0x1bcca80, 0x214d140})
        /usr/local/go/src/runtime/panic.go:884 +0x213
github.com/spidernet-io/spiderpool/pkg/multuscniconfig.generateNetAttachDef({0xc00053c910, 0x10}, 0xc0016e4580)
        /src/pkg/multuscniconfig/multusconfig_informer.go:400 +0x705
github.com/spidernet-io/spiderpool/pkg/multuscniconfig.(*MultusConfigController).syncHandler(0xc001132cc0, {0x216df08, 0xc000060038}, 0xc0016e4580)
        /src/pkg/multuscniconfig/multusconfig_informer.go:281 +0x197
github.com/spidernet-io/spiderpool/pkg/multuscniconfig.(*MultusConfigController).processNextWorkItem.func1({0x1adbfe0?, 0xc000650a50})
        /src/pkg/multuscniconfig/multusconfig_informer.go:214 +0x34e
github.com/spidernet-io/spiderpool/pkg/multuscniconfig.(*MultusConfigController).processNextWorkItem(0xc001132cc0)
        /src/pkg/multuscniconfig/multusconfig_informer.go:246 +0x5a
github.com/spidernet-io/spiderpool/pkg/multuscniconfig.(*MultusConfigController).runWorker(...)
        /src/pkg/multuscniconfig/multusconfig_informer.go:175
k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1(0x30?)
        /src/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:157 +0x3e
k8s.io/apimachinery/pkg/util/wait.BackoffUntil(0x1a9f960?, {0x2151e00, 0xc0007d8000}, 0x1, 0xc00051b560)
        /src/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:158 +0xb6
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0x0?, 0x3b9aca00, 0x0, 0x0?, 0x442ec5?)
        /src/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:135 +0x89
k8s.io/apimachinery/pkg/util/wait.Until(0x0?, 0xc000656240?, 0x0?)
        /src/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:92 +0x25
created by github.com/spidernet-io/spiderpool/pkg/multuscniconfig.(*MultusConfigController).Run
        /src/pkg/multuscniconfig/multusconfig_informer.go:166 +0x18c
panic: assignment to entry in nil map [recovered]
        panic: assignment to entry in nil map

goroutine 441 [running]:
k8s.io/apimachinery/pkg/util/runtime.HandleCrash({0x0, 0x0, 0xc000650a50?})
        /src/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:56 +0xd7
panic({0x1bcca80, 0x214d140})
        /usr/local/go/src/runtime/panic.go:884 +0x213
github.com/spidernet-io/spiderpool/pkg/multuscniconfig.generateNetAttachDef({0xc00053c910, 0x10}, 0xc0016e4580)
        /src/pkg/multuscniconfig/multusconfig_informer.go:400 +0x705
github.com/spidernet-io/spiderpool/pkg/multuscniconfig.(*MultusConfigController).syncHandler(0xc001132cc0, {0x216df08, 0xc000060038}, 0xc0016e4580)
        /src/pkg/multuscniconfig/multusconfig_informer.go:281 +0x197
github.com/spidernet-io/spiderpool/pkg/multuscniconfig.(*MultusConfigController).processNextWorkItem.func1({0x1adbfe0?, 0xc000650a50})
        /src/pkg/multuscniconfig/multusconfig_informer.go:214 +0x34e
github.com/spidernet-io/spiderpool/pkg/multuscniconfig.(*MultusConfigController).processNextWorkItem(0xc001132cc0)
        /src/pkg/multuscniconfig/multusconfig_informer.go:246 +0x5a
github.com/spidernet-io/spiderpool/pkg/multuscniconfig.(*MultusConfigController).runWorker(...)
        /src/pkg/multuscniconfig/multusconfig_informer.go:175
k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1(0x30?)
        /src/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:157 +0x3e
k8s.io/apimachinery/pkg/util/wait.BackoffUntil(0x1a9f960?, {0x2151e00, 0xc0007d8000}, 0x1, 0xc00051b560)
        /src/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:158 +0xb6
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0x0?, 0x3b9aca00, 0x0, 0x0?, 0x442ec5?)
        /src/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:135 +0x89
k8s.io/apimachinery/pkg/util/wait.Until(0x0?, 0xc000656240?, 0x0?)
        /src/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:92 +0x25
created by github.com/spidernet-io/spiderpool/pkg/multuscniconfig.(*MultusConfigController).Run
        /src/pkg/multuscniconfig/multusconfig_informer.go:166 +0x18c
```

--- a/pkg/multuscniconfig/multusconfig_informer.go
+++ b/pkg/multuscniconfig/multusconfig_informer.go
@@ -342,6 +342,9 @@ func (mcc *MultusConfigController) syncHandler(ctx context.Context, multusConfig
 func generateNetAttachDef(netAttachName string, multusConf *spiderpoolv2beta1.SpiderMultusConfig) (*netv1.NetworkAttachmentDefinition, error) {
 	multusConfSpec := multusConf.Spec.DeepCopy()
 	anno := multusConf.Annotations
+	if anno == nil {
+		anno = make(map[string]string)
+	}
 
 	var plugins []interface{}
 

